dist/js/materialize.js

```diff
@@ -220,8 +217,15 @@ jQuery.extend( jQuery.easing,
-    if (typeof define === 'function' && define.amd) {
-        define(['jquery', 'hammerjs'], factory);
+    if (typeof define === 'function' && define.amd == false) {
+        // hammerjs will already be publicly available
+        define(['jquery'], function($, Hammer) {
```

Comment:
Interesting, although I use require.js every day I never ran into that issue.
However, I'm looking forward to seeing your solution improving materialize.
Thanks in advance!